lib/epochtalk_server/bbc_parser.ex

```diff
@@ -0,0 +1,74 @@
+defmodule EpochtalkServer.BBCParser do
+  use GenServer
+  require Logger
+  alias Porcelain.Process, as: Proc
+  @timeout 10_000
```

Comment:
timeout's unreasonably long.  should reduce it to a realistic amount

long timeout will hang and back up the queue if there is an issue

Reply:
an example is when empty string is given as input